turbo/src/main/kotlin/dev/hotwire/turbo/fragments/TurboBottomSheetDialogFragment.kt

```diff
@@ -45,6 +45,13 @@ abstract class TurboBottomSheetDialogFragment : BottomSheetDialogFragment(),
      * Use [onViewCreated] for code touching
      * the Fragment's view and [onCreate] for other initialization.
      */
+    @Deprecated(
+        message = "Deprecated in Java",
+        replaceWith = ReplaceWith(
+            "super.onActivityCreated(savedInstanceState)",
+            "com.google.android.material.bottomsheet.BottomSheetDialogFragment"
+        )
+    )
```

Comment:
Curious, what's the purpose of adding these manual deprecations? They're marked as final and the existing deprecation is suppressed since they can't be used in consumer apps.